diff -Naur a/conf.py b/conf.py
--- a/conf.py	2018-09-15 13:59:24.000000000 +0200
+++ b/conf.py	2018-09-15 14:00:37.003799832 +0200
@@ -358,7 +358,7 @@
     registerGlobalValue(network, 'nick', ValidNickOrEmpty('', _("""Determines
         what nick the bot will use on this network. If empty, defaults to
         supybot.nick.""")))
-    registerGlobalValue(network, 'ident', ValidNickOrEmpty('', _("""Determines
+    registerGlobalValue(network, 'ident', registry.String('', _("""Determines
         the bot's ident string, if the server doesn't provide one by default.
         If empty, defaults to supybot.ident.""")))
     registerGlobalValue(network, 'user', registry.String('', _("""Determines
